src/module.cpp

```diff
@@ -771,29 +821,31 @@ cell AMX_NATIVE_CALL ezhttp_steam_to_steam64(AMX* amx, cell* params)
     return 1;
 }
 
-RequestId SendRequest(AMX* amx, RequestMethod method, OptionsId options_id, const std::string& url, const std::string& callback)
+RequestId SendRequest(AMX* amx, RequestMethod method, OptionsId options_id, const std::string& url, const std::string& callback, cell* data, const int data_len)
 {
     int callback_id = -1;
     if (!callback.empty())
     {
-        callback_id = MF_RegisterSPForwardByName(amx, callback.c_str(), FP_CELL, FP_DONE);
+        callback_id = MF_RegisterSPForwardByName(amx, callback.c_str(), FP_CELL, FP_ARRAY, FP_DONE);
         if (callback_id == -1)
         {
             MF_LogError(amx, AMX_ERR_NATIVE, "Callback function \"%s\" is not exists", callback.c_str());
             return RequestId::Null;
         }
     }
 
-    auto on_complete = [callback_id](RequestId request_id) {
+    auto on_complete = [callback_id, data, data_len](RequestId request_id) {
         if (callback_id == -1)
         {
             g_EasyHttpModule->DeleteRequest(request_id, true);
             return;
         }
 
-        MF_ExecuteForward(callback_id, request_id);
+        MF_ExecuteForward(callback_id, request_id, MF_PrepareCellArray(data, data_len));
         MF_UnregisterSPForward(callback_id);
 
+        delete data;
+
         g_EasyHttpModule->DeleteRequest(request_id, true);
     };
 
```

Comment:
I think it should be noted in README that with presence/absence of parameter in ezhttp_get,... callback function will be different.